kolibri/plugins/learn/assets/src/views/CurrentlyViewedResourceMetadata.vue

```diff
@@ -256,6 +253,16 @@
         }
       },
     },
+    $tr: {
```

Comment:
Seems like these strings are redundant, or if they aren't, we should be changing references to them to read from `learnString`?